### DISCUSSION TITLE
How to get diagnostic information from 'optimizer'?

### DISCUSSION THREAD

**Discussion by huytruong1908** [yalmip/YALMIP#1364] [General discussion] [answered]
Hello,
I would like to get diagnostic information such as yalmiptime or solvertime when using 'optimizer', is that possible? I know I can get this information when using 'optimize', but is this feature implemented for 'optimizer'?

**Comment by johanlofberg** [comment, marked as answer]
The 6th output returns everything available

**Reply by huytruong1908** [reply]
ah I get it now, thank you so much
